pageserver/src/tenant/mgr.rs

```diff
@@ -803,8 +801,9 @@ pub(crate) async fn set_new_tenant_config(
     // API to use is the location_config/ endpoint, which lets the caller provide
     // the full LocationConf.
     let location_conf = LocationConf::attached_single(new_tenant_conf, tenant.generation);
+    let tenant_shard_id = TenantShardId::unsharded(tenant_id);
```

Comment:
```suggestion
    // TODO(sharding): ...
    let tenant_shard_id = TenantShardId::unsharded(tenant_id);
```

Reply:
We may never update these APIs for sharding: I'd like us to cut over to controlling everything with /location_conf rather than updating the old APIs.  Just need to figure that out before going live with it.